src/inc-lib-form.php

```diff
@@ -71,11 +71,8 @@ function lovd_checkDBID ($aData)
     $nHasDBID = $_DB->query('SELECT COUNT(id) FROM ' . TABLE_VARIANTS . ' WHERE `VariantOnGenome/DBID` = ? AND id != ?', array($aData['VariantOnGenome/DBID'], $nIDtoIgnore))->fetchColumn();
     if ($nHasDBID && (!empty($sGenomeVariant) || !empty($aTranscriptVariants))) {
         // This is the standard query that will be used to determine if the DBID given is correct.
-        $sSQL = 'SELECT DISTINCT t.geneid, ' .
-                'CONCAT(IFNULL(vog.`VariantOnGenome/DNA`, ""), ";", IFNULL(GROUP_CONCAT(vot.`VariantOnTranscript/DNA` SEPARATOR ";"), "")) as variants, ' .
-                'vog.`VariantOnGenome/DBID` ' .
+        $sSQL = 'SELECT count(*) ' .
```

Comment:
Be careful :wink:
Replacing a normal select with a count(*) will change the way the results are returned, so then also handle that change in the fetching of the results. This change introduced a bug, where any variant can be assigned any given DBID of any other variant. Fixed in 665d952.